File: scripts/task_tested_with_compile_script/palindrome-detection/python/palindrome-detection.py
def is_palindrome(s):
  return s == s[::-1]

r = 1
def executeTask(i):
	b = is_palindrome("saippuakivikauppias" + `i`)
	return i

for i in xrange(2000000000):
	r = executeTask(i)

File: scripts/task_tested_with_compile_script/palindrome-detection/javascript/palindrome-detection.js
function isPalindrome(str) {
  return str === str.split("").reverse().join("");
}

var r = 1;

function executeTask(i) {
	var a = isPalindrome("saippuakivikauppias"+i);
	return i;
}

for ( var i = 0; i < 2000000000; ++i) {
	r = executeTask(i);
}
